core/templates/pages/collection-editor-page/services/collection-editor-state.service.ts

```diff
@@ -186,6 +188,7 @@ angular.module('oppia').factory('CollectionEditorStateService', [
             _collectionIsBeingSaved = false;
             if (successCallback) {
               successCallback();
+              $rootScope.$applyAsync();
```

Comment:
Since, this should be called anyway, maybe move it out of the `if` block?

Reply:
Done.